### ISSUE TITLE
Release dry-auto_inject 0.x version with support for ruby 3.2.1

### ISSUE THREAD

**Issue by radarek** [dry-rb/dry-auto_inject#89]
This commit https://github.com/dry-rb/dry-auto_inject/commit/5605781e1f11ec20b4d15cd9863d0fe40d4b4440 fixes dry-auto_inject for ruby 3.2.1 It was released as `1.0.1` version. It would be nice to release 0.x version with the same fix so it works with ruby 3.2.1 too. I have a project which can't update `dry-auto_inject` easily to 1.0.1 (karafka 1.x blocks updating it) and it would help to have `0.x` having this fix.

Currently I have to monkey patch `PASS_THROUGH` array.

**Comment by solnic** [comment]
Would you be able to cherry-pick this commit to latest release-0.x branch and open a PR for it? I could release it assuming all tests are green.

**Comment by radarek** [comment]
Sure, no problem. Should I bump to `0.9.1` or `0.10.0` version?

**Comment by solnic** [comment]
@radarek no, just cherry-pick the commit and open a PR against release-0.9.

**Comment by flash-gordon** [comment]
@radarek is this actual? 😅

**Comment by radarek** [comment]
@flash-gordon I don't need it anymore. I'm closing the ticket.
